runner/runner.go

```diff
@@ -47,22 +45,17 @@ type Command struct {
 }
 
 func (c Command) String() string {
-	var b bytes.Buffer
-	fmt.Fprintf(&b, "Command\n\tSnapshotID:\t%s\n\tArgv:\t%q\n\tTimeout:\t%v\n\tJobID:\t\t%s\n\tTaskID:\t\t%s\n",
-		c.SnapshotID,
-		c.Argv,
-		c.Timeout,
-		c.JobID,
-		c.TaskID)
+	s := fmt.Sprintf("Command -- SnapshotID:%s # Argv:%q # Timeout:%v # JobID:%s # TaskID:%s",
```

Comment:
I'm guessing this is to make these more usable in log lens at the expense of log file usefulness?